docs/source/notebooks/Image_Tut.ipynb

```diff
@@ -0,0 +1,468 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Image Classification with Cleanlab, PyTorch & Skorch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial will use Cleanlab to find potential label errors in the MNIST handwritten digit image dataset. This dataset contains 60,000 images of handwritten digits from 0 to 9. We'll find *more than a thousand* highly suspicious examples with label errors and edge cases by the end of this tutorial. \n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:**\n",
+    "\n",
+    "- Build a simple PyTorch neural net and wrap it with Skorch to make it scikit-learn compatible.\n",
+    "\n",
+    "- Compute the cross-validated predicted probabilities, ``pyx``, sometimes known as the out-of-sample predicted probabilities.\n",
+    "\n",
+    "- Generate a list of potential label errors with Cleanlab's ``get_noise_indices``.\n",
+    "\n",
+    "- Build and train a more robust model than the vanilla model after adapting it with Cleanlab's ``LearningWithNoisyLabels``. \n",
+    "\n",
+    "**Data:** https://www.openml.org/d/554"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Install the required dependencies"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "``%%capture`` is a magic function to hides the cell's output."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%capture\n",
+    "\n",
+    "%pip install cleanlab pandas matplotlib torch torchvision skorch"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Fetch and scale the MNIST dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.datasets import fetch_openml\n",
+    "\n",
+    "# Fetch the MNIST dataset\n",
+    "mnist = fetch_openml('mnist_784')\n",
+    "\n",
+    "# Assign the data features and labels to variable X and y respectively \n",
+    "X = mnist.data.astype('float32')\n",
+    "y = mnist.target.astype('int64')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "Bringing Your Own Data (BYOD)?\n",
+    "\n",
+    "Assign your data's features to variable ``X`` and its labels to variable ``y`` instead.\n",
+    "\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "# Scale the features which range from 0 to 255 (256 levels on the grayscale) \n",
+    "X /= 255.0\n",
+    "\n",
+    "# Then convert it to a numpy array\n",
+    "X = X.to_numpy()"
+   ]
+  },
```

Comment:
I'd merge this cell with the first one. That way a user who wants to use their own image dataset instead simply replaces that whole cell of the notebook.

Reply:
done!